doc/source/extending-jupyterhub.rst

```diff
@@ -176,3 +176,26 @@ report the current ``Ready/NotReady`` status of all nodes in the cluster.
      size for the workshop. This workflow also helps you avoid scrambling on
      the workshop day to set up the cluster and JupyterHub.
    - **After the workshop:** The cluster can be deleted.
+
+
+Setting up HTTPS
+----------------
+
+Zero to JupyterHub makes setting up HTTPS quite easy.
+
+1. Buy a domain name from a registrar. Pick whichever one you want
+2. Create an `A record` from the domain you want to use, pointing to the
+   external IP provided to the `proxy-public` service.
+3. Wait for the change to propagate. Propagation can take several minutes to
+   several hours. Wait till you can type in the name of the domain you bought,
+   and it shows you the hub.
+4. Add the following to your ``config.yaml``:
+
+   ..code-block:: yaml
+
+     proxy:
+       hosts:
+         - <your-domain-name>
+
```

Comment:
Also need 
```
https:
  auto:
    letsEncryptContactEmail: <your-email>
```

Which is validated upon attempting to deploy, but isn't included in this README

Reply:
ooo, great catch! ty!

Did this work for you otherwise?

Reply:
Looks like the hub is in a `CrashLoopBackOff` state with `v0.4` of the `k8s-hub` image.

(I just took the stock `values.yaml` from this branch to try to make it work).

My attention is being pulled by a fire, but I assume this is the same thing as before where I just need to put in a more recent hub image.